### DISCUSSION TITLE
Node Animation

### DISCUSSION THREAD

**Discussion by Hkp07** [JacquesLucke/animation_nodes#1883] [Q&A]
I have Done Animation Using noads but I want to change Sequence order of some parts.. so please tell me how to Do it...

**Comment by OmarEmaraDev** [comment]
Can you share the file and a more detailed description of what you have and what you are after?

**Reply by Hkp07** [reply]
Do I need to share blender file ?

**Reply by OmarEmaraDev** [reply]
Or a simplified version of it to demonstrate the issue.

**Reply by Hkp07** [reply]
hear is the blender [file...]
https://drive.google.com/folderview?id=1DO-UrAuF-yparNurH3MaeVgBlaVb2En6

**Reply by OmarEmaraDev** [reply]
You can just order the objects as described here:
https://docs.animation-nodes.com/release_notes/v2_0/id_keys/#selection-order-method

**Reply by Hkp07** [reply]
Okay.. I'll try to do it.... Thanks for helping me...

**Reply by Hkp07** [reply]
> You can just order the objects as described here: https://docs.animation-nodes.com/release_notes/v2_0/id_keys/#selection-order-method

Hey ... I Tried to do it.. Just because of I am biggner I am not able to do it... Could you please help me with process steps or something...

**Reply by OmarEmaraDev** [reply]
@Hkp07 I recommend you as this question here https://blender.stackexchange.com/.

**Reply by Hkp07** [reply]
Hey If it is possible I want to Learn... So can you provide any lesson Crash course??

**Reply by OmarEmaraDev** [reply]
See [here](https://docs.animation-nodes.com/) for a list of tutorials and courses.

**Reply by Hkp07** [reply]
Thank you For Replying... Can you just tell me How to do This kind of Animation-https://youtube.com/shorts/4W3YEWhqblg?feature=share

Just Because of I am biggner. I want proper guidance to make animation.

**Reply by OmarEmaraDev** [reply]
This animation is too long and complex for us to provide a guide for, as it would take too much time. We can help with a short and simple animation sequence, however.

**Reply by Hkp07** [reply]
I don't want full Animation Guide but I want to know How parts are coming. Do parts are following a path or it was done by using Node.

**Reply by OmarEmaraDev** [reply]
There is definitely a lot of manual work involved here. This could just be manually animated for all we know.
But this definitely wasn't done procedurally using nodes in the sense of being automated.
